scripts/examples/leviatom/query_nodes.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+curl -s http://localhost:8001/QueryNodes -X POST -H "Content-Type: application/json" -d "{\"period\":1,\"numRank\":100}"
+curl -s http://localhost:8002/QueryNodes -X POST -H "Content-Type: application/json" -d "{\"period\":1,\"numRank\":100}"
```

Comment:
Wanna a check?

Reply:
done, also need to change the floating point precision calculation in noderank. @jdyuankai